core/rust/qdbr/Cargo.toml

```diff
@@ -19,10 +19,11 @@ async-trait = "0.1.81"
 clap = "4.5.9"
 tokio = "1.38.0"
 log = "0.4.22"
+fail = "0.5"
```

Comment:
nit: is it used?

Reply:
Referenced here: https://github.com/questdb/questdb/pull/4830/files#diff-8d04401f1cc51365fe3e32f019cd720135ba920a1a7da7f19e9c9208478701fcR47